app/v2/models/response/retrieveBsas/foreignProperty/IncomeBreakdown.scala

```diff
@@ -27,15 +27,15 @@ case class IncomeBreakdown(rentIncome: Option[BigDecimal],
 
 object IncomeBreakdown {
   val nonFhlReads: Reads[IncomeBreakdown] = (
-    (JsPath \ "totalRentsReceived").readNullable[BigDecimal] and
+    (JsPath \ "rent").readNullable[BigDecimal] and
       (JsPath \ "premiumsOfLeaseGrant").readNullable[BigDecimal] and
       (JsPath \ "otherPropertyIncome").readNullable[BigDecimal] and
       (JsPath \ "foreignTaxTakenOff").readNullable[BigDecimal] and
       (JsPath \ "specialWithholdingTaxOrUKTaxPaid").readNullable[BigDecimal]
```

Comment:
`removal of foreignTaxTakenOff and specialWithholdingTaxOrUKTaxPaid` missed this requirement